### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
I# n this milestone I completed the following 

    

- [x] Created a table named owners with the following columns:
    - id: integer (set it as autoincremented PRIMARY KEY)
    - full_name: string
    - age: integer
- [x] Created a table named species with the following columns:
    - id: integer (set it as autoincremented PRIMARY KEY)
    - name: string
- [x] Modify animals table:
    - Make sure that id is set as autoincremented PRIMARY KEY
    - Remove column species
    - Add column species_id which is a foreign key referencing species table
    - Add column owner_id which is a foreign key referencing the owners' table
 - Insert new data into the owners' table
 - Insert new data into the species table
 - Modify inserted animals so it includes the species_id value
 - Modify inserted animals to include owner information (owner_id)
 - Write queries (using JOIN) to answer some questions

## Built With 🖤 

 - SQL

_Thanks In advance for your Feedback 💚_

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -9,3 +9,17 @@ CREATE TABLE animals (
 	weight_kg numeric NOT NULL,
 	species VARCHAR(50)
 );
+
+CREATE TABLE owners (
+    owner_id SERIAL,
+	full_name VARCHAR(100) NOT NULL,
+	age INT NOT NULL,
+	PRIMARY KEY (id_owner)
```
Comment (AshakaE) [inline schema.sql]: This primary key `id_owner` does not exist I believe you mean `owner_id` 

- [x] Kindly update query

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @Hamzaoutdoors  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Hamzaoutdoors) [reply schema.sql]: yes it's my bad I didn't copy it correctly, Thank you for your comment

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉 

## Hi @Hamzaoutdoors, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**